src/v/resource_mgmt/storage.h

```diff
@@ -243,6 +243,116 @@ class disk_space_manager {
+         * necessary to meet the configured usage target.
+         */
+        void set_available_reclaimable(size_t size) noexcept {
+            _available_reclaimable = size;
+        }
+
+        /*
+         * The total amount of data that can be safely reclaimed and which if
+         * reclaimed would not violate the local retention policy.
+         */
+        void set_local_retention_reclaimable(size_t size) noexcept {
+            _local_retention_reclaimable = size;
+        }
+
+        /*
+         * The amount of data usage above the target threshold. when space
+         * management runs if this value is positive then space management will
+         * try to ensure this much data is reclaimed, if that is by applying
+         * normal retention policy or applying space management over policy.
+         */
+        void set_target_excess(size_t size) noexcept { _target_excess = size; }
+
+        /*
+         * The amount of data estimated to be reclaimed by the currently
+         * installed space management schedule that is above the local retention
+         * policy setting.
+         */
+        void set_reclaim_local(size_t size) noexcept { _reclaim_local = size; }
+
+        /*
+         * The amount of data estimated to be reclaimed by the currently
+         * installed space management schedule that is above the non-hinted
+         * low-space threshold setting.
+         */
+        void set_reclaim_low_non_hinted(size_t size) noexcept {
+            _reclaim_low_non_hinted = size;
+        }
+
+        /*
+         * The amount of data estimated to be reclaimed by the currently
+         * installed space management schedule that is above the hinted
+         * low-space threshold setting.
+         */
+        void set_reclaim_low_hinted(size_t size) noexcept {
+            _reclaim_low_hinted = size;
+        }
+
+        /*
+         * The amount of data estimated to be reclaimed by the currently
+         * installed space management schedule that is above the active segment.
+         */
+        void set_reclaim_active_segment(size_t size) noexcept {
+            _reclaim_active_segment = size;
+        }
+
+        /*
+         * The amount of data that the latest reclaim schedule has estimated
+         * will be reclaimed. A positive value means that space management is
+         * trying to reclaim data that would othewise not be reclaimed by the
+         * normal retention policy.
+         */
+        void set_reclaim_estimate(size_t size) noexcept {
+            _reclaim_estimate = size;
+        }
+
+    private:
+        disk_space_manager* _sm;
+        metrics::internal_metric_groups _metrics;
+        size_t _total_usage{0};
+        size_t _retention_reclaimable{0};
+        size_t _available_reclaimable{0};
+        size_t _local_retention_reclaimable{0};
+        size_t _target_excess{0};
+        size_t _reclaim_local{0};
+        size_t _reclaim_low_non_hinted{0};
```

Comment:
some of these are unused/not exported to prometheus. why?

Reply:
Hmm, good spot. It looks like these last few weren't wired up. Perhaps I dropped a commit in the rebase? Anyway, I'll find it. Thanks!

Reply:
fixed